packages/dx-react-grid/src/plugins/virtual-table/virtual-table-state.tsx

```diff
@@ -75,9 +75,12 @@ class VirtualTableStateBase extends React.PureComponent<VirtualTableStateProps,
       getRows(newPageIndex, loadCount);
 
       const virtualRowsCache = trimRowsToInterval(virtualRows, newBounds);
-      const newRowCount = infiniteScrolling
-        ? Math.max(newBounds.end + loadCount, stateAvailableCount)
-        : totalRowCount;
+      const newRowCount = getAvailableRowCount(
+        infiniteScrolling,
```

Comment:
`infiniteScrolling` -> `isInfiniteScrolling` ?

Reply:
newRowCount -> nextRowCount ?
But we should use only one word for variables. `next` / `new`

Reply:
This `infiniteScrolling` comes from props. 
We can do: `{..., infiniteScrolling: isInfiniteScrolling, ... } = this.props`, but should we?

Reply:
@ushkal,

Maybe we should change the prop name too?

Reply:
> We can do: {..., infiniteScrolling: isInfiniteScrolling, ... } = this.props, but should we?

I think we should

Reply:
prop name was coordinated with techwriters and I don't see why the current name is bad. Besides that, renaming prop is a bc